package.json

```
@@ -0,0 +1,45 @@
+{
+  "name": "axe-linter-action",
+  "version": "1.0.0",
+  "scripts": {
+    "prebuild": "rimraf dist",
+    "build": "ncc build src/index.ts --license licenses.txt",
+    "lint": "eslint .",
+    "fmt": "prettier --write \"src/**/*.{js,ts,tsx,json,md,css,html}\"",
```

No need to manually specify a glob, Prettier will do it on its own:

```suggestion
    "fmt": "prettier --write .",
```